### PR TITLE
[Rector] Fixes rector notice use

### PR DESCRIPTION
**Pull request by samsonasik**
Cherry-pick from https://github.com/codeigniter4/CodeIgniter4/pull/4426/commits/abae0fd9bc92794995191f49c81e84fac08dcba6

**Checklist:**
- [x] Securely signed commits

### CODE REVIEW

File: system/Config/BaseService.php [hunk 1]
```diff
@@ -11,6 +11,20 @@
...
+use Config\Filters as ConfigFilters;
+use Config\Format as ConfigFormat;
+use Config\Honeypot as ConfigHoneyPot;
+use Config\Images;
+use Config\Migrations;
+use Config\Pager as ConfigPager;
+use Config\View as ConfigView;
+use CodeIgniter\HTTP\Request;
+use Config\Toolbar as ConfigToolbar;
+use Config\Validation as ConfigValidation;
```
Comment (paulbalandan) [inline system/Config/BaseService.php:27]: sort alpha

Comment (samsonasik) [reply system/Config/BaseService.php:27]: It seems already sortered:

```bash
➜  CodeIgniter4 git:(fix-rector-notice) ✗ php-cs-fixer fix system/Config/BaseConfig.php --rules=ordered_imports 
Loaded config default.
Using cache file ".php_cs.cache".
```

Comment (paulbalandan) [reply system/Config/BaseService.php:27]: Sort alphabetically. `Config\App` should come before `Config\Cache`. `CodeIgniter\HTTP\Request` is misplaced. `Config\View` is misplaced too.

Comment (samsonasik) [reply system/Config/BaseService.php]: fix

**Review by paulbalandan — APPROVED** [review]

**Pull request merged**